packages/react-utilities/src/hooks/useOnClickOutside.ts

```diff
@@ -5,7 +5,7 @@ export type UseOnClickOutsideOptions = {
   /**
    * The element to listen for the click event
    */
-  element?: Document;
+  element: Document;
```

Comment:
```suggestion
  element: Document | undefined;
```

We should allow explicit `undefined` as it's what we will get in SSR.